### PR TITLE
Span class and grammar standardization in messages

### PR DESCRIPTION
**Pull request by kingofkosmos**
If all messages in the game were consistent, it would soon become second nature to recognize quickly when something out of the ordinary happens in the message tab out of the corner of your eye, rather than having to constantly browse through rainbow colored messages.

Changed the messages in following situations with these ideas in mind:

When you have to hold still for more than 50 ticks: "spanclass='notice': You start doing stuff..."

You fail at doing something: "spanclass='warning': You fail at stuff!"

All the things you do always happen in present tense: "spanclass='notice': You do stuff."

You see someone do something normal plainly without spans: "[someone] does stuff."

You see someone do something alarming (attacking, emagging, proboscis etc.): "spanclass='warning': [someone] does stuff."

### CODE REVIEW

File: code/datums/diseases/transformation.dm [hunk 1]
```diff
@@ -189,7 +189,7 @@
 	stage2	= list("You are turning a little green.")
 	stage3	= list("<span class='danger'>Your limbs are getting oozy.</span>", "<span class='danger'>Your skin begins to peel away.</span>")
 	stage4	= list("<span class='danger'>You are turning into a slime.</span>")
-	stage5	= list("<span class='danger'>You have become a slime.</span>")
+	stage5	= list("<span class='danger'>You become a slime.</span>")
```
Comment (Incoming5643) [inline code/datums/diseases/transformation.dm]: the phrasing on these messages is a nethack reference

It's not a huge deal, I just wanted to point out that's why it's like that

Comment (kingofkosmos) [inline code/datums/diseases/transformation.dm]: Ok I was unaware of that, I'll change it back.

File: code/game/objects/items/devices/flash.dm [hunk 2]
```diff
@@ -66,12 +66,14 @@
 			M.confused += power
 			terrible_conversion_proc(M, user)
 			M.Stun(1)
-			user.visible_message("<span class='disarm'>[user] blinds [M] with the flash!</span>")
+			user.visible_message("<span class='disarm'>[user] blinds [M] with the flash!</span>", "<span class='danger'>You blind [M] with the flash!</span>")
+			M << "<span class='userdanger'>[user] blinds you with the flash!</span>"
```
Comment (kingofkosmos) [inline code/game/objects/items/devices/flash.dm]: Wait I just realized... Does M get two messages now? Like:
"[User] blinds [M]!"
"[User] blinds you!"?

I don't know how to test this on my own server.

**Comment by Cheridan** [comment]
Well the ship is already in the water now I guess but for the record it's better to do this kind of thing bit-by-bit rather than having over 200 lines changed in a PR.

**Comment by kingofkosmos** [comment]
Well shit. @phil235 told me it was ok!

**Comment by Jordie0608** [comment]
Pull from upstream master to your local branch; git will shout at you which files it couldn't merge, those are the conflicts.

Merge conflicts can't really be avoided.

**Comment by kingofkosmos** [comment]
Alright, I'll look at the merge conflicts. After I presumably handle those, should I divide this PR into 3 different PR's of 90 file changes each? Or how would this be best handled "bit-by-bit"?

**Comment by Cheridan** [comment]
Well like I said, the boat's already in the water so just fix the conflicts and then scream like hell for a maintainer to merge it.

**Comment by Miauw62** [comment]
pulling from upstream master is a pretty awful way to resolve conflicts but since nobody has a good guide on how to use rebases it works :V

File: code/game/machinery/autolathe.dm [hunk 3]
```diff
@@ -124,7 +124,7 @@
 		user << "<span class=\"alert\">The autolathe is full. Please remove glass from the autolathe in order to insert more.</span>"
 		return 1
 	if (O.m_amt == 0 && O.g_amt == 0)
-		user << "<span class=\"alert\">This object does not contain significant amounts of metal or glass, or cannot be accepted by the autolathe due to size or hazardous materials.</span>"
+		user << "<span class=\"alert\">This object does not contain significant amounts of metal or glass, or cannot be accepted by the autolathe due to size or hazardous materials</span>"
```
Comment (Cheridan) [inline code/game/machinery/autolathe.dm]: This one looks off.

File: code/modules/projectiles/guns/projectile/revolver.dm [hunk 4]
```diff
@@ -88,31 +88,31 @@
 	..()
 	if(istype(A, /obj/item/weapon/screwdriver))
 		if(magazine.caliber == "38")
-			user << "<span class='notice'>You begin to reinforce the barrel of [src].</span>"
+			user << "<span class='notice'>You begin to reinforce the barrel of [src]...</span>"
 			if(magazine.ammo_count())
 				afterattack(user, user)	//you know the drill
 				user.visible_message("<span class='danger'>[src] goes off!</span>", "<span class='danger'>[src] goes off in your face!</span>")
```
Comment (Cheridan) [inline code/modules/projectiles/guns/projectile/revolver.dm]: goes off in your face! should be userdanger

**Comment by Cheridan** [comment]
I made a wiki page on text formatting so that maybe people will actually use them properly in the future and not invalidate all this work

https://tgstation13.org/wiki/Text_Formatting

feel free to build onto it if you want to add anything

**Comment by kingofkosmos** [comment]
Wow, thanks, Cheridan!

**Comment by Cheridan** [comment]
panic

**Comment by phil235** [comment]
The merge conflict fixing readded files that were deleted in my telecoms PR.

**Comment by kingofkosmos** [comment]
Fuck. Sorry @phil235, the merge conflict stuff was kinda overwhelming.

**Comment by phil235** [comment]
Well, I don't really blame you, I even told you it was okay to make the whole change in one PR. I guess this shows why multiple smaller PRs is preferable.

**Comment by Miauw62** [comment]
eh, probably my fault too since I sort of guided you through doing the conflict fixing :P (and also told you that one big PR is ok)

these are pretty special circumstances anyway, code files dont get moved or removed often.

**Pull request merged**